circle_evolution/helpers.py

```diff
@@ -24,9 +25,14 @@ def load_target_image(image_path, color=cv2.COLOR_BGR2GRAY, size=None):
     """
     if not os.path.exists(image_path):
         raise FileNotFoundError(f"Image was not found at {image_path}")
-    img = cv2.imread(image_path)
-    # Performs Image Convertion
-    target = cv2.cvtColor(img, color)
+
+    if color:
+        target = cv2.imread(image_path, cv2.IMREAD_COLOR)
+        # Switch from bgr to rgb
+        target = target[:, :, (2, 1, 0)]
```

Comment:
That's definitely overkill but can we use the `cv2.cvtColor(im_cv, cv2.COLOR_BGR2RGB)` for this? I know that `cv2` works with BGR and `matplotlib` with RGB; also by the end of the day, `cv2` will just change the order of arrays just like you did, so that would be for the sake of consistency use of OpenCV - totally optional and up to you.